doc/adr/0006-no-certificat-pinning.md

```diff
@@ -0,0 +1,28 @@
+# 5. No certificate pinning
+
+Date: 2022-12-01
+
+## Status
+
+Accepted
+
+## Context
+
+During the identification flow, the eID-Client connects to the server of a Service Provider (eService) and 
+to an eID-Server. Therefor, a TLS connection is being established and the eID-Client validates the server identity 
```

Comment:
`Therefore`

Reply:
I used "Therefor" on purpose. Is that not how you use it? See [here](https://www.grammarly.com/blog/therefore-vs-therefor/#:~:text=Therefore%20is%20an%20adverb%20that,%E2%80%9D%20or%20%E2%80%9Cfor%20it.%E2%80%9D)

Reply:
Oh, I did not know that word existed.